packages/react-composites/src/composites/common/CaptionsBannerMoreButton.tsx

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IContextualMenuItem } from '@fluentui/react';
+import { ControlBarButtonProps, _StartCaptionsButton } from '@internal/react-components';
+import React from 'react';
+import { useMemo } from 'react';
+import { usePropsFor } from '../CallComposite/hooks/usePropsFor';
+import { buttonFlyoutIncreasedSizeStyles } from '../CallComposite/styles/Buttons.styles';
+import { useLocale } from '../localization';
+import { MoreButton } from './MoreButton';
+
+/** @private */
+export interface CaptionsBannerMoreButtonProps extends ControlBarButtonProps {
+  onCaptionsSettingsClick?: () => void;
+}
+
+/**
+ *
+ * @private
+ */
+export const CaptionsBannerMoreButton = (props: CaptionsBannerMoreButtonProps): JSX.Element => {
+  const localeStrings = useLocale();
+
+  const startCaptionsButtonProps = usePropsFor(_StartCaptionsButton);
+
+  const moreButtonStrings = useMemo(
+    () => ({
+      label: localeStrings.strings.call.moreButtonCallingLabel,
+      tooltipOffContent: localeStrings.strings.callWithChat.moreDrawerButtonTooltip
```

Comment:
Is it the more button on caption banner? Then we should create separated strings for this, anything string changes moreButton in controlbar shouldn't change this one

Reply:
Based on design control bar more button should be the exact same text as caption banner more button, so they should share the same string?

Reply:
No.. we should still allow users to override those strings one by one